### PR TITLE
[WIP] Onboarding AMO to prow

### PR DESCRIPTION
**Pull request by austinmartinh**

### CODE REVIEW

File: clusters/app.ci/.diff [hunk 1]
```diff
@@ -177,7 +218,37 @@ diff --recursive -u --new-file '--label=o' '--label=m' o m
    - openshift/odo
    - openshift/odo-init-image
    review_acts_as_lgtm: true
-@@ -2013,21 +2010,6 @@
+@@ -1883,29 +1864,6 @@
```
Comment (austinmartinh) [inline clusters/app.ci/.diff]: @laurafitzgerald

File: Makefile [hunk 2]
```diff
@@ -94,7 +94,6 @@ cluster-roles:
 
 roles: cluster-operator-roles cluster-roles
 .PHONY: roles
-
```
Comment (laurafitzgerald) [inline Makefile]: Hi @austinmartinh this line shouldn't be removed.

**Comment by laurafitzgerald** [comment]
@austinmartinh I left one comment in line but other than that the only other things to be done are to do the rebase and ask about the additional config that's getting added by `make jobs` We can do that one the other items are resolved.

**Comment by laurafitzgerald** [comment]
@austinmartinh you can close this one now that you have a new pr.

**Pull request closed without merging**